### DISCUSSION TITLE
What minimum version of Prisma is the tracing integration still supported?

### DISCUSSION THREAD

**Discussion by dankellett** [getsentry/support#58] [Support] [answered]
We are having trouble getting db traces from Prisma to show in the Prisma portal. As far as we can tell we have it setup as instructed, but it seems to be failing silently. We are running an older version of Prisma (2.23.0) due to a dependency on it and suspect this might be the issue.

It's a bit of a complicated stack, so the extra context might help: Apollo, Prisma, Nexus, Next.

**Comment by AbhiPrasad** [comment, marked as answer]
@dankellett We've been testing against Prisma `3.0.0` - so that's what we support. I'll update the docs to make this more clear.

If you enable the SDK with `debug: true` passed in as an option, you're able to see failures from Prisma. What do the debug logs show you?
